src/tools/rbd/CMakeLists.txt

```diff
@@ -51,15 +52,22 @@ set(rbd_srcs
   action/Trash.cc
   action/Watch.cc)
 
+if(CURSES_FOUND)
+  add_definitions(-DHAVE_CURSES)
```

Comment:
please don't add definitions globally if not necessary. might want to use `target_compile_definitions()` instead.